package-lock.json

```diff
@@ -1,13 +1,28 @@
 {
   "name": "ng-annotate-patched",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
Not that it matters much, but this change is a little strange I think, since there is no 1.7.0 yet?